docs/lake/design.md

```diff
@@ -141,23 +141,27 @@ in the Zed language is referred to as "this".
 
 The create command initiates a new pool with a single branch called `main`.
 
+> Zed lakes can be used without branches.  When referencing a pool without
+> a branch, the tooling presumes the "main" branch as the default, and everything
+> can be done on main without having to think about branching.
+
 ### Branch
 
-A branch is created from a parent branch (often "main") with the `branch`
-command, e.g.,
+A branch is simply a named pointer to a commit object in the zed lake.
```

Comment:
```suggestion
A branch is simply a named pointer to a commit object in the Zed lake.
```

The rest of the doc is consistent about capitalizing "Zed" when talking about the project and `zed` when mentioning the command line tool.